vocabularies/PersonalData.xml

```diff
@@ -181,14 +181,14 @@ Terms for contact and address information are defined in the [Communication voca
             </Record>
             <Record>
               <Annotation Term="Common.Experimental" />
-              <PropertyValue Property="Value" String="EndOfPurposeDate" />
-              <Annotation Term="Core.Description" String="The end of purpose date of a record" />
+              <PropertyValue Property="Value" String="PotentialEndOfBusinessDate" />
```

Comment:
```suggestion
              <PropertyValue Property="Value" String="EndOfBusinessDate" />
```
With the changed description, I see no need for the prefix "Potential" any more. Suggest to drop it, it is confusing.